cluster/manifests/event-logger/vpa.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: event-logger-vpa
+  namespace: kube-system
+  labels:
+    application: kubernetes
+    component: event-logger
+spec:
+  targetRef:
+    apiVersion: apps/v1
+    kind: StatefulSet
+    name: kubernetes-event-logger
+  updatePolicy:
+    updateMode: Auto
+  resourcePolicy:
+    containerPolicies:
+    - containerName: logger
+      minAllowed:
+        memory: {{.ConfigItems.event_logger_mem_min}}
```

```suggestion
        memory: {{.Cluster.ConfigItems.event_logger_mem_min}}
```

Just a pet peeve to have it consistent :sweat_smile: